index.rst

```diff
@@ -4,54 +4,146 @@
++------------------------------------+-------------------+
+| Secondary Obscuration Diameter     |  0.423m           |
++------------------------------------+-------------------+
+
+The optical design (in Zemax) for the telescope and LATISS instrument can be found in
+`Document-27702 <https://docushare.lsst.org/docushare/dsweb/Get/Document-27702>`_ (credentials required). The
+The original alignment was performed by Roberto Tighe using an alignment laser with the telescope at ~30 degrees
+elevation, a short report on the original first light (by eye) alignment is found
+`in Document-35621 <https://docushare.lsst.org/docushare/dsweb/Get/Document-35621>`_.
+
+.. 1.5Arcsec coma/mm is another interesting number that came out of the alignment
+
+Telescope collimation and focus can be done in a variety of ways. We opted to use CWFS techniques because the main
+camera for LSST uses this technique during standard operation.
+
+Implications and Limitations of the Technique
+=============================================
+
+The CWFS technique for collimation (and focus) is highly affected by seeing, particularly dome seeing. Experience
+has shown that long (~30s) exposures are required to average out the effects of dome seeing to have a stable solution.
+Seeing variations of ~20nm for the defocus and coma Zernikes is to be expected.
+
+The telescope has very good optics (0.06 waves RMS) and therefore image quality is fully seeing limited. The CWFS
+method measures the wavefront error (WFE) introduced by the optics and their alignment beyond the seeing limit
+therefore so long as the
+total WFE is at (or below) the diffraction limit of the system then optimizing beyond this point is of no use. Using
+a diffraction limit of lambda/4 for 632nm gives 158nm. Assuming all the WFE comes from the defocus through spherical
+aberration terms (4-11), then this allows each term to have a magnitude of $158/sqrt(8)$~55nm. Therefore, we
+consider any focus or coma abberation magnitude below 50nm to be acceptable and not require further optimization as
+the PSF will be heavily seeing dominated.
+
+One should note that the above technique is only an approximation and breaks down in certain cases. For example, if one
+has 158nm of coma and all over aberrations are zero then you will see very significant coma in the PSF! The ~50 nm
+guideline is meant as a guideline and is not a hard truth. In exceptional seeing conditions (which we've not
+encountered to date), the instrumental PSF may begin to have an effect. To date, the PSF quality is dominated by dome
+seeing and tracking error.
+
+In correcting the collimation/focus using the hexapod it is important to understand the limits of motion. Motion of the
+hexapod and how it relates to WFE correction is defined by a sensitivity matrix. The derivation of this matrix is
+discussed in detail in `this technote <tstn-016.lsst.io>`_. For the purposes of this document, a user must know how the
```

Comment:
I think that "in this tech note" leaves the impression that it is this tech note. Maybe rephrase to something like; 

> The derivation of this matrix is discussed in detail in `tstn-016 <https://tstn-016.lsst.io>`_.